src/vm/perfmap.cpp

```diff
@@ -159,8 +165,17 @@ void PerfMap::LogMethod(MethodDesc * pMethod, PCODE pCode, size_t codeSize)
     EX_CATCH{} EX_END_CATCH(SwallowAllExceptions);
 }
 
-// Log a native image to the map.
-void PerfMap::LogNativeImage(PEFile * pFile)
+
+void PerfMap::LogImageLoad(PEFile * pFile)
+{
+    if (s_Current != NULL)
+    {
+        s_Current->LogImage(pFile);
+    }
+}
+
+// Log an IL image load to the map.
```

Comment:
Please remove IL from the comment, as this can be any managed image (IL, NGEN, R2R).